src/data/reducers/assetsFilters.js

```diff
@@ -0,0 +1,20 @@
+import { FILTER_UPDATED } from '../actions/assets';
+
+const initialState = {
+  courseId: 'course-v1:edX+DemoX+Demo_Course',
+  page: 0,
+  images: false,
+  documents: false,
+  other: false,
```

Comment:
Because the URI doesn't have any filter params applied by default, you *could* get away with just not storing the `false` defaults at all -- one less place we'd have to hardcode these filter keys. Since these filter-specific parameters (`images`, `documents`, `other`) are closely related to each other, but not as related to `page` and `courseId`, I'd suggest nesting them into one arbitrary object in this reducer (e.g. `filters: {}`). Side benefit: when it's time to assemble the URI with multiple filter params in the api client, we can do something slick like
```
Object.keys(assetFilters.filters).reduce((memo, key) => {
    assetFilters.filters[key] && memo.push(assetFilters.filters[key]);
    return memo;
}).join(',');
```
@MichaelRoytman and I messed around with something similar last week.

Reply:
How about:

```javascript
const initialState = {
  courseId: 'course-v1:edX+DemoX+Demo_Course',
  page: 0,
  pageSize: 50,
  assetTypes: {},
  sort: 'sort',
};
```
If we ever add another filter set to the table, we could just add another key and object to this structure.

Reply:
LGTM!